server/store/oci/oci.go

```diff
@@ -333,249 +325,111 @@ func (s *store) Delete(ctx context.Context, ref *corev1.RecordRef) error {
 	}
 }
 
-// PushSignature stores OCI signature artifacts for a record.
+// PushSignature stores OCI signature artifacts for a record using cosign attach signature and uploads public key to zot for verification.
 func (s *store) PushSignature(ctx context.Context, recordCID string, signature *signv1.Signature) error {
 	logger.Debug("Pushing signature artifact to OCI store", "recordCID", recordCID)
```

Comment:
consider moving the logic for signature under `server/store/oci/sign_store.go` file to separate from the common storage interface for OCI